.circleci/config.yml

```diff
@@ -77,9 +77,7 @@ jobs:
           name:  Run E2E test
           command: >-
               export KUBECONFIG=$GOPATH/src/github.com/kubermatic/machine-controller/.kubeconfig &&
-              go test -race -tags=e2e -parallel 240 -v -timeout 30m  ./test/e2e/... -identifier=$CIRCLE_BUILD_NUM ||
-              (kubectl logs -n kube-system $(kubectl get pods -n kube-system|egrep '^machine-controller'|
```

Comment:
Should we upload the logs to a private gcp bucket?

Reply:
Maybe? I don't find it extremely helpful tbh.

Reply:
Then we'll add uploading to a bucket when we see the need for it